debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/PostgresTestResourceLifecycleManager.java

```diff
@@ -33,6 +33,7 @@
             .withEnv("POSTGRES_DB", POSTGRES_DBNAME)
             .withEnv("POSTGRES_INITDB_ARGS", "-E UTF8")
             .withEnv("LANG", "en_US.utf8")
+            .withExposedPorts(POSTGRES_PORT)
```

Comment:
As you are touching this class: there's this new concept of "Dev Services" in Quarkus now, which spins up required resources such as a database automatically when detecting configuration hinting at it. So e.g. Postgres will be started when using the PG Hibernate dialect. This means this custom lifecycle manager may not be needed any longer, provided we can customize the Dev Service PG in the way we needed. Do you want to take a look at this and remove this class if possible?

Reply:
Sure I can take a look.